modules/installation-guide/partials/proc_installing-che-on-kubernetes_using_chectl_and_helm.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-google-cloud-platform
+// installing-{prod-id-short}-on-aws
+// installing-{prod-id-short}-on-microsoft-azure
+
+
+[id="installing-{prod-id-short}-on-kubernetes-using-{prod-cli}_and_helm_{context}"]
+= Installing {prod-short} on {k8s-platform} using {prod-cli}
+
+.Prerequisites
+
+* The `{prod-cli}` management tool is available. See xref:using-the-chectl-management-tool.adoc[].
+* The `helm` tool is available, with version 2.15 or higher. See link:https://helm.sh/[Helm].
+
+.Procedure
+
+* To install {prod-short} on {k8s-platform}, run the following `{prod-cli}` command:
++
+[subs="+attributes"]
+----
+$ {prod-cli} server:deploy --installer=helm --platform=k8s --domain={domain} --multiuser
+› Current Kubernetes context: 'minikube'
+  ✔ Verify Kubernetes API...OK
+  ✔ 👀  Looking for an already existing {prod} instance
+    ✔ Verify if {prod} is deployed into namespace "{prod-namespace}"...it is not
+  ✔ ✈️  {kubernetes} preflight checklist
+    ✔ Verify if kubectl is installed
+    ✔ Verify remote kubernetes status...done.
+    ✔ Check {kubernetes} version: Found v1.15.12-gke.2.
+    ✔ Verify domain is set...set to {domain}.
+    ↓ Check if cluster accessible [skipped]
+  ✔ Following {prod} logs
+    ↓ Start following Operator logs [skipped]
+    ✔ Start following {prod} server logs...done
+    ✔ Start following Postgres logs...done
+    ✔ Start following Keycloak logs...done
+    ✔ Start following Plugin registry logs...done
+    ✔ Start following Devfile registry logs...done
+    ✔ Start following namespace events...done
+  ✔ 🏃‍  Running Helm to install {prod}
+    ✔ Verify if helm is installed
+    ✔ Check Helm Version: Found v3.4.1+gc4e7485
+    ✔ Create Namespace ({prod-namespace})...does already exist.
+    ✔ Check {prod} TLS certificate...self-signed TLS certificate secret found
+    ✔ Check Cluster Role Binding...does not exists.
+    ✔ Preparing {prod} Helm Chart...done.
+    ✔ Updating Helm Chart dependencies...done.
+    ✔ Deploying {prod} Helm Chart...done.
+  ✔ ✅  Post installation checklist
+    ✔ PostgreSQL pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ Devfile registry pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ Plugin registry pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ {prod} pod bootstrap
+      ✔ Scheduling...done
+      ✔ Downloading images...done
+      ✔ Starting...done
+    ✔ {prod} status check...done
+  ✔ Prepare post installation output...done
+  ✔ Show important messages
+    ✔ {prod} {prod-ver} has been successfully deployed.
+    ✔ Documentation             : https://www.eclipse.org/che/docs/
```

Comment:
@themr0c Could you tell if we have a variable for documentation URL Che/CRW?

Reply:
We don't have one and it's a good idea to add one. Please define it as a new `asciidoc.attribute` in `antora-playbook.yml`:  ```prod-url: https://www.eclipse.org/che/docs```

Reply:
will do

Reply:
I've added prod-docs-url, since prod-url already exists and point to the Che Server